Frontend/ui-customer/src/Screens/Handlers.purs

```diff
@@ -31,7 +31,7 @@ import Screens.PermissionScreen.Handler (permissionScreen) as UI
 import Screens.ReferralScreen.Handler (referralScreen) as UI
 import Screens.SavedLocationScreen.Handler (savedLocationScreen) as UI
 import Screens.SelectLanguageScreen.Handler (selectLanguageScreen) as UI
-import Screens.SplashScreen.Handler (splashScreen) as UI
+import Screens.SplashScreen.Handler (toggleSplash) as UI
```

Comment:
we can remove this from this list. Having a separate function  makes it easier to handle. I tried to keep it in `Utils.purs` but it is causing cyclic dependency. so decided to keep in Handler. we will remove from this ReExport.

Reply:
- Having a separate function makes it easier to handle

Makes sense but here I'm trying to unifying the flow file so it will clash with that. Flow file will only call screen handlers, within that if we want to have functions we can have it

Reply:
Got it. You are suggesting to keep the function in flow file itself. So that it will internally call handler as other flow functions does.

Reply:
Yup

Reply:
Done